src/main/java/seedu/address/storage/JsonAdaptedRoom.java

```diff
@@ -0,0 +1,37 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.room.Room;
+
+public class JsonAdaptedRoom {
+
+    private int roomNumber;
+    private boolean isOccupied;
+    private String patient;
+    private String task;
```

Comment:
We can add a TODO here since we are not using the `Patient` and `Task` in this class yet.